index.js

```diff
@@ -1,27 +1,66 @@
-const mongoose = require('mongoose');
-
-// Import of the model Recipe from './models/Recipe.model.js'
-const Recipe = require('./models/Recipe.model');
-// Import of the data from './data.json'
-const data = require('./data');
-
-const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
-
-// Connection to the database "recipe-app"
-mongoose
-  .connect(MONGODB_URI, {
-    useCreateIndex: true,
-    useNewUrlParser: true,
-    useUnifiedTopology: true
-  })
-  .then(self => {
-    console.log(`Connected to the database: "${self.connection.name}"`);
-    // Before adding any documents to the database, let's delete all previous entries
-    return self.connection.dropDatabase();
-  })
-  .then(() => {
-    // Run your code here, after you have insured that the connection was made
-  })
-  .catch(error => {
-    console.error('Error connecting to the database', error);
-  });
+const mongoose = require('mongoose');
+
+// Import of the model Recipe from './models/Recipe.model.js'
+const Recipe = require('./models/Recipe.model');
+// Import of the data from './data.json'
+const data = require('./data');
+
+const MONGODB_URI = 'mongodb://localhost:27017/recipe-app';
+
+// Connection to the database "recipe-app"
+mongoose
+  .connect(MONGODB_URI, {
+    useCreateIndex: true,
+    useNewUrlParser: true,
+    useUnifiedTopology: true
+  })
+  .then(self => {
+    console.log(`Connected to the database: "${self.connection.name}"`);
+    // Before adding any documents to the database, let's delete all previous entries
+    // return self.connection.dropDatabase();
+  })
+  .then(() => {
+    // Run your code here, after you have insured that the connection was made
+
+    Recipe.create({

```

Comment:
You could save the object inside a variable before the promise and then just add the variable inside the promise. Helps with readability. Also before you are using the first element of the data.json you could just target that via bracket notation. like data[0]